src/components/createRecipe/recipeForm/Step1.jsx

```diff
@@ -29,8 +31,8 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-const getAllCategoiresUrl = "http://localhost:3333/api/category";
-const getAllTagsUrl = "http://localhost:3333/api/tag";
+const getAllCategoiresUrl = `${process.env.REACT_APP_API_BASE_URL}api/category`;
+const getAllTagsUrl = `${process.env.REACT_APP_API_BASE_URL}api/tag`;
 
```

Comment:
Why insert this here? As we talked about, the baseURL is inserted by the axios functions in utils/axios.js. Better to import them!

Reply:
Material ui takes dropdown component's `listUrl` attribute only takes a string and executes a `get` request to display the options. 
In this instance, we don't need to import the Axios function